crates/ruff_linter/resources/test/fixtures/flake8_pyi/PYI062.py

```diff
@@ -0,0 +1,9 @@
+from typing import Literal
+
+x: Literal[True, False, True, False]  # PYI062 twice here
+
+y: Literal[1, print("hello"), 3, Literal[4, 1]]  # PYI062 on the last 1
```

Comment:
Can you add test cases where the inner literal is entirely redundant? e.g.

`Literal[1, Literal[1]]`
`Literal[1, 2, Literal[1, 2]]`
`Literal[1, Literal[1], Literal[1]]`
`Literal[1, Literal[2], Literal[2]]`
`Literal[1, Literal[2, Literal[1]]]`

Reply:
These will be more important for when we add a fix, I think.

Reply:
It's also good to have coverage for cases like `Literal[1, 1, 1]` where we should raise the diagnostic twice.